src/i18n/es/index.ts

```diff
@@ -57,16 +61,31 @@ const es = {
 	language: 'Idioma',
 	lastUsedModels: 'Modelos utilizados recientemente',
 	light: 'Claro',
+	lowVram: 'Bajo VRAM',
+	mainGpu: 'GPU principal',
```

Comment:
I agree. I've translated them because I wanted to keep the UI's language consistent, but it's true that these terms are usually shown in english no matter the language of the speaker.